package.json

```diff
@@ -18,16 +18,16 @@
     "bin": "bin"
   },
   "scripts": {
-    "build-docs": "scripts/build/build-docs.sh",
-    "build-schema": "scripts/build/build-schema.sh",
-    "build-wiki": "scripts/build/build-wiki.sh",
-    "publish-wiki": "scripts/publish/publish-wiki.sh",
-    "publish-docs": "scripts/publish/publish-docs.sh",
-    "test": "scripts/test/test.sh",
-    "test-integration": "scripts/test/test-integration.sh",
-    "test-infra": "scripts/test/test-infra.sh",
-    "test-browser": "scripts/test/test-browser.sh",
-    "test-lint": "scripts/test/test-lint.sh"
+    "build-docs": "node npm/build-docs.js",
```

Comment:
Can you add a system/repository test to ensure all scripts begin with `node `? @kunagpal (ps also add in other repos)